docs/tutorials/fundamentals/features.md

```diff
@@ -0,0 +1,62 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+(features)=
+# Useful features
```

Comment:
Perhaps change to title to **Features** without Useful. Then add a 1-2 sentences on what this page covers: command palette, features table, ...